### DISCUSSION TITLE
Modifiers in the JSON format are entirely different from Modifiers for CSV formats

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/hospital-price-transparency#73] [Q&A] [answered]
In the JSON format the way of representing modifiers is very different from the CSV formats.  What can be represented is very different.  Shouldn't the formats just be different textual representations of the same information?  Right now one can't represent the same thing in the JSON format that one could in the CSV wide format.

Modifiers in the JSON format are at the root level and seem to apply across all items ("Standard Charge Information Objects").  Also modifiers in the JSON format aren't associated with a separate numeric charge instead they have a textual description of how the modifier adjusts the charges. In the JSON example the modifier description is: "150% payment adjustment for the item or service to which the modifier is appended".  Because of this placement of the modifiers in the JSON hierarchy, they also have to repeat the payer and plan name to associate the modification with a specific payer-plan.  But they can't be associated with a specific item (set of codes and code types).

In contrast, the CSV wide format has a field for plural "modifiers" and one can have separate rows for each item with and without modifiers and have separate charges on each of these rows.  So, the modifiers can only be on specific items and the adjustments can vary by item.  This is very different from the JSON version.  The CSV version is very similar to the Transparency in Coverage rule.

Am I mis-reading the specification?  

If I am not, then this seems like a serious problem with the JSON format.

**Comment by knh762002** [comment]
A response to this post could also answer our questions from Post #22 

As the JSON schema stands now, I am also having to encode modifiers to the additional_generic_notes object for Code Types, such as CDM, that may not have payer negotiated charges associated with it.

A quick response would be greatly appreciated.

Thank you!

**Reply by nmayle** [reply]
I would suggest that you add modifiers as a "hospital created data element" to the "Standard Charge Object".
The CSV templates explicitly allow "hospital created data elements"; they aren't mentioned in the JSON format, but there doesn't seem to be a reason you couldn't add a modifiers field there where it logically makes sense and is parallel to the CSV format.
Giving it a reasonable field name instead of just using the comment field.
Perhaps others will follow your lead.
I hope we get an answer from the CMS or CMS-related folks.

**Comment by RidenourK** [comment]
@nmayle and @knh762002. Thank you for your questions and suggestions.
 
Please refer to lines 446-463 of the JSON examples for an example of how to encode modifiers in JSON.

Additionally, yes, you may add modifiers as a "hospital created data element" to the "Standard Charge Object" in JSON if you so choose.

**Reply by nmayle** [reply]
I understand the current specification and see it in the example.  There are two broad issues with this design:

* it doesn't match the CSV templates, so anyone using the data will have to store and interpret data from the CSV templates differently from data from the JSON templates.  
* it doesn't match how modifiers change charges: the gross charge is also typically adjusted, the modifiers don't necessarily apply uniformly to all codes with the same adjustment percentage, the representation has the adjustment as a free form string of text when if the modifiers were put in a better place in template, we could get actual numeric charge values

**Comment by RidenourK** [comment, marked as answer]
@nmayle. Thank you for your suggestion that we consider future Hospital Price Transparency – Data Dictionary GitHub repository updates to align the CSV and JSON templates with regards to modifiers.

**Reply by nmayle** [reply]
We also see some hospitals putting the modifiers in the JSON template in the Standard Charge Information Object so it is parallel to the codes.

BTW: the description of the JSON format doesn't mention the use of hospital created data elements.  Are they allowed in any object or just some...

**Reply by RidenourK** [reply]
@nmayle As indicated in the CY 2024 OPPS/ASC Final Rule, nothing precludes a hospital from offering additional context and information to the public in its MRF, so long as the MRF conforms to the formatting requirements required at § 180.50(c)(2). Please refer to [88 FR 82095](https://www.federalregister.gov/d/2023-24293).

**Comment by angelina-abitino** [comment]
Would CMS consider updating the JSON schema to have a modifier attribute added to the standard_charges object? It could contain a list of modifier codes and would match the existing CSV schema.

If providers start posting files in this way using a "hospital created data element", without there being a schema definition giving standard rules for how the element should be handled, it will not be possible to process this field when aggregating many MRFs in a unified database.
